### DISCUSSION TITLE
Help Plz

### DISCUSSION THREAD

**Discussion by jorTurtleGaming** [Loukious/TikTokStreamKeyGenerator#27] [Q&A]
I am brand spanking new to Python (literally didn't hear about it until searching for this plugin) and I'm trying my darnedest to get this to work but I'm like a fish trying to climb a tree. Is there a video that would be helpful, or a walkthrough from absolute-start-to-finish for getting this up and running?

**Comment by Loukious** [comment]
Download the .exe file from [here](https://github.com/Loukious/TikTokStreamKeyGenerator/releases/download/v1.0.3/TiktokStreamKeyGenerator.exe) and launch it and follow the steps in the readme. You don't need to know about Python to use it.

**Reply by jorTurtleGaming** [reply]
THANK YOU, YOU'RE A LEGEND!

**Reply by jorTurtleGaming** [reply]
Any chance you know what the regions correlate to?

**Reply by Loukious** [reply]
> Any chance you know what the regions correlate to?

It's where your stream will be recommended too. They also use your IP so it won't be 100% exclusive to the region you choose.

**Reply by jorTurtleGaming** [reply]
It looks to me like they know I'm "tricking the system" because any time I use the tool to go live, I get like 15 views for a 4 hour stream, but if I use Tiktok Live Studio I get like 500 after 20 minutes, lol. Thought maybe the region might have something to do with it but I guess not.

**Reply by Loukious** [reply]
> It looks to me like they know I'm "tricking the system" because any time I use the tool to go live, I get like 15 views for a 4 hour stream, but if I use Tiktok Live Studio I get like 500 after 20 minutes, lol. Thought maybe the region might have something to do with it but I guess not.

Should be fixed.

**Comment by jbkings15** [comment]
for the game tag id what that means i play warzone so what do i put??

**Comment by jbkings15** [comment]
![image](https://github.com/user-attachments/assets/e1aee59c-31c1-4553-b2de-85bbe10bb2b7)
what this error means i only get it when i select gaming for my topic???

**Reply by xPinM** [reply]
Hi. I have the same error. let me know if you find a solution thanks
